src/options.rs

```diff
@@ -108,3 +101,21 @@ impl TryFrom<&Args> for Options {
         })
     }
 }
+
+fn build_glob_set(glob_set: &Vec<String>) -> Result<Option<GlobSet>>{
```

Comment:
looks like this needs `cargo fmt` (or CI will fail)

Reply:
Actually CI will not yet reject it because it's not hooked up #67

Reply:
yes, this was my bubu, I will execute fmt (and also clippy) once everything else is processed :)